android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarDelegate.kt

```diff
@@ -1,20 +1,20 @@
 package com.builttoroam.devicecalendar
 
 import android.Manifest
-import android.graphics.Color;
 import android.annotation.SuppressLint
 import android.content.ContentResolver
 import android.content.ContentUris
 import android.content.ContentValues
 import android.content.Context
 import android.content.pm.PackageManager
 import android.database.Cursor
+import android.graphics.Color
 import android.net.Uri
 import android.provider.CalendarContract
 import android.provider.CalendarContract.CALLER_IS_SYNCADAPTER
 import android.provider.CalendarContract.Events
 import android.text.format.DateUtils
-import com.builttoroam.devicecalendar.common.Constants
+import com.builttoroam.devicecalendar.common.*
```

Comment:
Will this import all the nested constants as well eg Constants.Companion.ATTENDEE_EMAIL_INDEX.
If so, can we remove the individual imports below

Reply:
Fixed